src/pyasm/web/html_wdg.py

```diff
@@ -353,22 +353,18 @@ def add_gradient(my, name, palette_key, modifier=0, range=-20, reverse=False, de
 
 
 
-    def add_border(my, modifier=None, style=None, direction=None, color=None, size=None):
+    def add_border(my, modifier=None, style="solid", direction=None, color="border", size="1px"):
         '''@params: direction can be top left bottom right, default to None''' 
-        if not style:
-            style = "solid"
-        style_name = 'border'
+        border_direction = 'border'
```

Comment:
since we have border_direction and color both iniitalized to be 'border'. a comment should be added here to explain at least the color variable saying it could be an color name, hex color, or a palette attribute..